modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/IdentityProxy.java

```diff
@@ -108,7 +121,8 @@ public void getToken(Context contextt, TokenCallBack call, String clientID, Stri
 		}
 	}
 
-	private void getStoredToken() throws InterruptedException, ExecutionException, TimeoutException {
+	private void getStoredToken()
```

Comment:
Do we not need to specifically address "InterruptedException" here in the implementation? For example, there might be instances where the task needs to be continued even though there's an interruption, etc. So, we need to think carefully as to how we should handle this.